.travis.yml

```diff
@@ -7,12 +7,19 @@ services:
 language: cpp
 
 env:
-  matrix:
-    - COMPILER=gcc;   LCG_RELEASE=LCG_96b; STANDARD=17; COMPILER_VERSION=gcc8; PYTHON_CHECK="no"
-    - COMPILER=gcc;   LCG_RELEASE=LCG_96python3; STANDARD=17; COMPILER_VERSION=gcc8; PYTHON_CHECK="no"
-    - COMPILER=clang; LCG_RELEASE=LCG_96b; STANDARD=17; COMPILER_VERSION=clang8; PYTHON_CHECK="no"
-    - COMPILER=gcc;   LCG_RELEASE=LCG_96b; STANDARD=14; COMPILER_VERSION=gcc62; PYTHON_CHECK="no"
-    - PYTHON_CHECK="yes"; COMPILER=gcc;   LCG_RELEASE=LCG_96b; STANDARD=17; COMPILER_VERSION=gcc8;
+  - COMPILER=gcc;   LCG_RELEASE=LCG_96b; STANDARD=17; COMPILER_VERSION=gcc8; PYTHON_CHECK="no"
+  - COMPILER=gcc;   LCG_RELEASE=LCG_96b; STANDARD=17; COMPILER_VERSION=gcc8; PYTHON_CHECK="no"; CMAKE_ARGS="-DBUILD_SHARED_LIBS=OFF"
+  - COMPILER=gcc;   LCG_RELEASE=LCG_96python3; STANDARD=17; COMPILER_VERSION=gcc8; PYTHON_CHECK="no"
```

Comment:
```suggestion
  - COMPILER=gcc;   LCG_RELEASE=LCG_96bpython3; STANDARD=17; COMPILER_VERSION=gcc8; PYTHON_CHECK="no"
```

Reply:
I have not previously changed this, because the python3 stack was not published till recently